src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -121,19 +121,27 @@ class SyncControllerSpec extends FlatSpec with Matchers {
     peerMessageBus.reply(MessageFromPeer(NodeData(Seq(stateMptLeafWithAccount)), peer2.id))
     peerMessageBus.expectMsg(Unsubscribe(MessageClassifier(Set(NodeData.code), PeerSelector.WithId(peer2.id))))
 
+    //wait for peers throttle
+    Thread.sleep(200.milliseconds.toMillis)
```

Comment:
I meant that we could refer to the value from configuration rather than hard-coding it.
I also pointed out that we probably don't need `time.advance` if we already do the `sleep`.